src/plugins/intel_cpu/src/utils/ngraph_utils.hpp

```diff
@@ -47,5 +49,28 @@ inline bool isDynamicNgraphNode(const std::shared_ptr<const ngraph::Node>& op) {
     return ret;
 }
 
+inline std::string get_port_name(const ov::Output<const ov::Node>& port, const bool is_legacy_api) {
+    std::string name;
+    // TODO: Should use tensor name as the port name, but many legacy tests still use legacy name
+    // plus sometimes it will get empty tensor name.
+    if (!is_legacy_api) {
+        name = {};
+        // auto tensor_ptr = port.get_tensor_ptr();
+        // if (tensor_ptr->get_names().size() >= 1) {
+        //     name = port.get_tensor_ptr()->get_any_name();
+        // }
+    }
+    if (name.empty()) {
+        bool is_input = ov::op::util::is_parameter(port.get_node());
+        if (is_input) {
+            name = ov::op::util::get_ie_output_name(port);
+        } else {
+            const auto node = port.get_node_shared_ptr();
+            name = ov::op::util::get_ie_output_name(node->input_value(0));
```

Comment:
is not this function universal and cannot be used instead of get_port_name ?

Reply:
It is suggested that plugin API 2.0 should use tensor_name to replace legacy tensor_name or friendly_name (implement by `ov::op::util::get_ie_output_name`),  and cpu plugin need a unified helper method to align port_name getter both for cpu plugin interface and cpu's  graph class.
`ov::op::util::get_ie_output_name` also is exported to python API, if we change it, there will bring many failures, it need another ticket to solve such problem.